### PR TITLE
cmake: Using Zephyr CMake package for locating Zephyr installation

### PR DESCRIPTION
**Pull request by tejlmand**
This commit fixes an issue in building NCS docs on windows where the
Zephyr base could contain backslashes.

This is fixed by using the new Zephyr CMake package.

Signed-off-by: Torsten Rasmussen <Torsten.Rasmussen@nordicsemi.no>

### CODE REVIEW

**Comment by greg-fer** [comment]
This is a doc framework request, strictly speaking, so while I'm really grateful for your work and help, I don't think my approval is required here. However, if this change warrants any modification to the existing build instructions (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_build.html), kindly please include me in the related PR.

**Comment by tejlmand** [comment]
> This is a doc framework request, strictly speaking, so while I'm really grateful for your work and help, I don't think my approval is required here.

As you discovered the initial issue, your approval will be appreciated as a confirmation that this PR fixes the issue you reported on your machine.

> However, if this change warrants any modification to the existing build instructions (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_build.html), kindly please include me in the related PR.

Good point, will update those.

**Comment by tejlmand** [comment]
> >    However, if this change warrants any modification to the existing build instructions (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_build.html), kindly please include me in the related PR.
>
> Good point, will update those.

Done

File: doc/nrf/doc_build.rst [hunk 1]
```diff
@@ -62,30 +61,17 @@ Complete the following steps to build the documentation output:
...
-        .. code-block:: console
-
-           ..\..\..\zephyr\zephyr-env.cmd
-   * On Linux or macOS:
+#. Generate the Ninja build files:
 
         .. code-block:: console
 
-           source ../../../zephyr/zephyr-env.sh
+           cmake -GNinja -B_build .
```
Comment (ru-fu) [inline doc/nrf/doc_build.rst:68]: Why are we changing this command? The Zephyr instructions (at least in our fork) for building the docs haven't changed.

And so zephyr-env.sh isn't needed anymore? Or not supported anymore?
That probably means https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html#setting-up-the-command-line-build-environment must be changed as well.

Comment (tejlmand) [reply doc/nrf/doc_build.rst:68]: > 
> 
> Why are we changing this command? The Zephyr instructions (at least in our fork) for building the docs haven't changed.
> 


> And so zephyr-env.sh isn't needed anymore? Or not supported anymore?

It is not needed anymore.
Going to send a PR upstream as well, cause it's also no longer needed upstream.
But it is still supported.

> That probably means https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html#setting-up-the-command-line-build-environment must be changed as well.

Yes, and that will be updated as part of this:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/2232
where samples will be updated.

**Comment by greg-fer** [comment]
@tejlmand , also the additional fix is working now.

**Pull request merged**